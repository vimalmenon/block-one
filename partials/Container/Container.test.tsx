import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import { Container } from './'

describe('Container Testing', () => {
  test('Container : Snapshot testing', () => {
    const wrapper = shallow(
      <Container>
        <div>content</div>
      </Container>
    )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
