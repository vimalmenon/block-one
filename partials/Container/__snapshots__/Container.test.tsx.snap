// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Testing Container : Snapshot testing 1`] = `
<div
  className="flex flex-col h-screen bg-gray-100"
>
  <main
    className="flex flex-col h-full"
  >
    <Header />
    <section
      className="flex flex-grow justify-center"
    >
      <div
        className="flex container"
      >
        <div>
          content
        </div>
      </div>
    </section>
    <Footer />
  </main>
</div>
`;
